internal/driver/nodeserver_helpers.go

```diff
@@ -372,33 +357,19 @@ func (ns *NodeServer) prepareLUKSVolume(ctx context.Context, devicePath string,
 // Returns an error if any operation fails during the process.
 func (ns *NodeServer) closeLuksMountSource(ctx context.Context, volumeID string) error {
 	log := logger.GetLogger(ctx)
-	log.V(4).Info("Entering closeLuksMountSource", "volumeID", volumeID)
 
-	mountSource, err := ns.getMountSource(ctx, volumeID)
+	volumeName, err := ns.getMountSource(ctx, volumeID)
 	if err != nil {
-		return errInternal("closeLuksMountSource failed to get mount source %s: %v", volumeID, err)
+		return fmt.Errorf("Could not get volumename in pvcxxxxxxxxx format from given volumeID: %s", volumeID)
```

Comment:
This error message is not necessarily true. We just split the volumeID string into 2 parts but delimiter `-`. We aren't checking if the second string has `pvc` as a prefix. 

Reason: CSI-sanity tests send requested to CSI driver that does not conform with `pvc` prefix format

Reply:
reverted